BullsAndCowsTest/BullsAndCowsGameTest.cs

```diff
@@ -8,10 +9,120 @@ public class BullsAndCowsGameTest
         [Fact]
         public void Should_create_BullsAndCowsGame()
         {
-            var secretGenerator = new SecretGenerator();
+            var secretGenerator = new TestSecretGenerator();
             var game = new BullsAndCowsGame(secretGenerator);
             Assert.NotNull(game);
             Assert.True(game.CanContinue);
         }
+
```

Comment:
test for secretGenerate and validinput check should be included

Reply:
yes, it is really important